plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheMutableTransactionLog.java

```diff
@@ -0,0 +1,117 @@
+        assertUpdate("CALL system.flush_metadata_cache(schema_name => CURRENT_SCHEMA, table_name => 'test_transaction_log_not_cached')");
+        assertFileSystemAccesses(
+                "SELECT * FROM test_transaction_log_not_cached",
+                ImmutableMultiset.<CacheFileSystemTraceUtils.CacheOperation>builder()
+                        .addCopies(new CacheFileSystemTraceUtils.CacheOperation("InputFile.length", "00000000000000000002.checkpoint.parquet"), 2)
+                        .addCopies(new CacheFileSystemTraceUtils.CacheOperation("Input.readTail", "00000000000000000002.checkpoint.parquet"), 2)
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("InputFile.newStream", "00000000000000000003.json"))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("InputFile.newStream", "_last_checkpoint"))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.readCached", "key=p1/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.readCached", "key=p2/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Input.readFully", "key=p1/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Input.readFully", "key=p2/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.writeCache", "key=p1/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.writeCache", "key=p2/", 0, 220))
+                        .build());
+        assertFileSystemAccesses(
+                "SELECT * FROM test_transaction_log_not_cached",
+                ImmutableMultiset.<CacheFileSystemTraceUtils.CacheOperation>builder()
+                        .addCopies(new CacheFileSystemTraceUtils.CacheOperation("InputFile.length", "00000000000000000002.checkpoint.parquet"), 2)
+                        .addCopies(new CacheFileSystemTraceUtils.CacheOperation("Input.readTail", "00000000000000000002.checkpoint.parquet"), 2)
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("InputFile.newStream", "00000000000000000003.json"))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("InputFile.newStream", "_last_checkpoint"))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.readCached", "key=p1/", 0, 220))
+                        .add(new CacheFileSystemTraceUtils.CacheOperation("Alluxio.readCached", "key=p2/", 0, 220))
+                        .build());
+    }
+
+    private void assertFileSystemAccesses(@Language("SQL") String query, Multiset<CacheFileSystemTraceUtils.CacheOperation> expectedCacheAccesses)
+    {
+        assertUpdate("CALL system.flush_metadata_cache()");
+        DistributedQueryRunner queryRunner = getDistributedQueryRunner();
+        queryRunner.executeWithPlan(queryRunner.getDefaultSession(), query);
```

Comment:
Instead of `queryRunner.getDefaultSession()` we could use `queryRunner.getSession()`

Reply:
@Praveen2112  I can't seem to find the getSession method in the QueryRunner interface or its implementations. Is this method recent? I just rebased on master but can't find it.

Reply:
Sorry I meant `getSession` from `AbstractTestQueryFramework`

Reply:
Done!